server/src/test/java/io/crate/expression/scalar/formatting/ToCharFunctionTest.java

```diff
@@ -62,6 +62,11 @@ public void testEvaluateInterval() {
         assertEvaluate("to_char(INTERVAL '1 year 2 months 3 weeks 5 hours 6 minutes 7 seconds', 'YYYY MM DD HH12:MI:SS')", "0001 03 22 05:06:07");
     }
 
+    @Test
+    public void testEvaluateIntervalMilliseconds() {
```

Comment:
very minor: camelCase is used only in old tests. 
We use underscore (`test_evaluate_interval_milliseconds`) in new tests. Same applies for other camelCase instances

Reply:
Fixed.